main.js

```diff
@@ -139,36 +139,25 @@ function startApp() {
     }
   }
 
-  function handleUnhandledException(err) {
-    bt.report(err, {}, handleFinishedReport);
-  }
-
-  if (pjson.env === 'production') {
-    bt.initialize({
-      disableGlobalHandler: true,
-      endpoint: 'https://streamlabs.sp.backtrace.io:6098',
-      token: 'e3f92ff3be69381afe2718f94c56da4644567935cc52dec601cf82b3f52a06ce',
-      attributes: {
-        version: pjson.version,
-        processType: 'main'
-      }
-    });
+  Raven.config('https://6971fa187bb64f58ab29ac514aa0eb3d@sentry.io/251674', {
+    release: process.env.SLOBS_VERSION, 
+    sampleRate: 0.1
```

Comment:
Let's actually kill the sample rate here.  I may regret this but... Main process exceptions should be extremely rare and low volume (they are fatal to the application) so let's just record everything.

Reply:
Removed it!